examples/file-upload-download/src/controllers/file-download.controller.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-file-upload-download
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {inject} from '@loopback/context';
+import {get, HttpErrors, param, Response, RestBindings} from '@loopback/rest';
+import fs from 'fs';
+import path from 'path';
+import {promisify} from 'util';
+
+const readdir = promisify(fs.readdir);
+
+const SANDBOX = path.resolve(__dirname, '../../.sandbox');
+
+/**
+ * A controller to handle file downloads using multipart/form-data media type
+ */
+export class FileDownloadController {
+  @get('/file-download', {
```

Comment:
nitpick: the name is a bit confusing, maybe change it to something like '/list-files'?

Reply:
I wanted to make it symmetric with `file-upload`. But I'm open.

Reply:
We can even make all endpoints as `/files/*`:

- POST /files (upload)
- GET /files (list files)
- GET /files/{fileName} (download files)

Reply:
Adjusted.

Reply:
great 👍